### PR TITLE
provide examples of usage extendReducer()

### PR DESCRIPTION
**Pull request by belkosyak**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -100,4 +100,100 @@ A reducer which returns `null` as the initial state.
...
+
+Lets imagine that we have two separate counters and they should provide different behaviours
+on each tick that occurs. One have should only increment its own by 1 and another one should
+also change sign of a number it holds. And you have two separate widgets with such a counters,
+which works independently and starts with different numbers.
+
+```js
+import { compose, combineReducers } from 'redux';
+import { extendReducer, initialReducer } from 'redux-reuse';
+import { ignoreActions } from 'redux-ignore';
```
Comment (xamfoo) [inline README.md]: unused :)

File: README.md [hunk 2]
```diff
@@ -100,4 +100,100 @@ A reducer which returns `null` as the initial state.
...
+  combineReducers({
+    smoothCounter: increment(tickActionType)(initialReducer(startingNumber)),
+    jumpingCounter: compose(
+      increment(tickActionType),
+      changeSign(tickActionType)
+    )(initialReducer(startingNumber))
+  });
+
+const app = combineReducers({
+  widget1: widgetReducer(WIDGET1_TICK, 0),
```
Comment (xamfoo) [inline README.md:140]: Should we declare or import the action types somewhere in the example?

File: README.md [hunk 3]
```diff
@@ -100,4 +100,100 @@ A reducer which returns `null` as the initial state.
...
+  [CHANGE_SIGN]: changeSign,
+};
+
+const counter = (state = { modes: {}, value: null }, { payload: mode }) => {
+  switch (action.type) {
+    case ADD_MODE:
+      state.modes[mode] = true;
+      break;
+    case REMOVE_MODE:
+      delete state.modes[mode];
```
Comment (xamfoo) [inline README.md]: Is it ok to mutate the state here?

File: README.md (hunk 2 above)
Comment (belkosyak) [inline README.md:140]: fixed

File: README.md (hunk 3 above)
Comment (belkosyak) [inline README.md]: Redux does not require you to use only immutable data structures, so I think yes, it is ok for simplicity

**Pull request merged**